packages/spec-api/src/types.ts

```diff
@@ -80,20 +61,21 @@ export interface ServiceRequestFile {
 }
 
 export interface ServiceRequest {
-  body?: any;
+  body?: MockBody | MockMultipartBody;
   status?: number;
   /**
    * Query parameters to match to the request.
    */
+  query?: Record<string, unknown>;
+  /**
+   * Path parameters to match to the request.
+   */
```

Comment:
`params` is to generic, why not `path` or `pathParam`?

Reply:
yeah name was terrible, forgot to remove this one, it's not used